microsoft-edge/progressive-web-apps-chromium/how-to/shortcuts.md

```diff
@@ -13,24 +13,24 @@ keywords: progressive web apps, PWA, Edge, Windows, Taskbar, jumplist, shortcut
 
 App shortcuts let users accomplish common tasks more quickly and easily and can increase their engagement with an app.
 
-On mobile devices, shortcuts can usually be accessed by long-pressing an app icon. On Windows, shortcuts are integrated as Jumplists. Jumplists define popup menus that appear when you use the context menu (right-click) on a tile in the Start Menu or an icon in the Taskbar.
+On mobile devices, shortcuts can usually be accessed by long-pressing an app icon.  On Windows, shortcuts are integrated as Jumplists.  Jumplists define popup menus that appear when you right-click a tile in the Start menu, or right-click an icon in the taskbar.
 
-:::row:::
-    :::column span="1":::
-        :::image type="content" source="../media/edge-ios-shortcuts.png" alt-text="Shortcuts on Microsoft Edge on iOS." lightbox="../media/edge-ios-shortcuts.png":::
-    :::column-end:::
-    :::column span="1":::
-        :::image type="content" source="../media/pwa-shortcuts-in-taskbar.png" alt-text="Jumplist on the Twitter app on Windows." lightbox="../media/pwa-shortcuts-in-taskbar.png":::
-    :::column-end:::
-:::row-end:::
+The next figure shows shortcuts on Microsoft Edge on iOS:
 
-PWAs can also define common tasks as shortcuts in the [Web App Manifest](./web-app-manifests.md).
+:::image type="content" source="../media/edge-ios-shortcuts.png" alt-text="Shortcuts on Microsoft Edge on iOS.":::
+
+<!-- remove/replace screenshot? -->
+The next figure shows a jumplist on the Twitter app on Windows:
+
+:::image type="content" source="../media/pwa-shortcuts-in-taskbar.png" alt-text="Jumplist on the Twitter app on Windows.":::
```

Comment:
The screenshot is actually not showing the Twitter app at all. I had replaced it with the Webboard demo app, but forgot to change the alt-text at the time.
So the right thing to do here is change the sentence you added to something like "The next figure shows a jumplist on the Webboard app on Windows:", and change the alt-text accordingly.